app/model/db/idx_token.py

```diff
@@ -135,7 +136,10 @@ class IDXBondToken(TokenBase):
     # Purpose
     purpose = Column(String(2000))
     # Memo
-    memo = Column(String(2000))
+    if engine.name == "mysql":
+        memo = Column(Text)
+    else:
+        memo = Column(String(10000))
```

Comment:
If use MySQL, previous migration files that expand varchar length to 10000 will cause error message like below, so memo field should be changed to `Text`.

> Row size too large. The maximum row size for the used table type, not counting BLOBs, is 65535. This includes storage overhead, check the manual. You have to change some columns to TEXT or BLOBs